functions.go

```diff
@@ -80,16 +80,23 @@ func EfgsUploadKeys(w http.ResponseWriter, r *http.Request) {
 	efgs.UploadBatch(w, r)
 }
 
-// EfgsDownloadKeys downloads EFGS keys batch
-func EfgsDownloadKeys(ctx context.Context, m pubsub.Message) error {
-	return efgs.DownloadAndSaveKeys(ctx, m)
+// EfgsDownloadKeys downloads EFGS keys - most recent batch
+func EfgsDownloadKeys(w http.ResponseWriter, r *http.Request) {
+	if err := efgs.DownloadAndSaveKeys(r.Context()); err != nil {
+		http.Error(w, fmt.Sprintf("Error: %v", err), 500)
+	}
 }
 
 // EfgsDownloadYesterdaysKeys downloads EFGS keys batch from whole yesterday
 func EfgsDownloadYesterdaysKeys(w http.ResponseWriter, r *http.Request) {
 	efgs.DownloadAndSaveYesterdaysKeys(w, r)
 }
 
+//EfgsImportKeys Imports given keys
+func EfgsImportKeys(ctx context.Context, m pubsub.Message) error {
```

Comment:
The `EfgsImportKeys` function will never be deployed. Its definition in the `deploy.yaml` is missing.